Covid-19 _Real-time_Notification/Covid.py

```diff
@@ -0,0 +1,39 @@
+from plyer import notification
+import requests
+from bs4 import BeautifulSoup
+import time
+
+def notify_user(title, message):
+    notification.notify(
+    title = title,
+    message = message,
+    app_icon = "./Covid-19_Real-time_Notification/Notify_icon.ico" ,
+    timeout = 3)
```

Comment:
take this time input from user

Reply:
I guess it is better to not take this as user input or else it will just lead to confusion between both time for the user